src/main/kotlin/com/openlattice/hazelcast/serializers/RemoveMembersFromOrganizationAssemblyProcessorStreamSerializer.kt

```diff
@@ -29,15 +29,14 @@ import com.openlattice.assembler.processors.RemoveMembersFromOrganizationAssembl
 import com.openlattice.hazelcast.StreamSerializerTypeIds
 import com.openlattice.organizations.SecurablePrincipalList
 import org.springframework.stereotype.Component
-import javax.inject.Inject
 
 @Component
 class RemoveMembersFromOrganizationAssemblyProcessorStreamSerializer
     : SelfRegisteringStreamSerializer<RemoveMembersFromOrganizationAssemblyProcessor>,
       AssemblerConnectionManagerDependent<Void?> {
     private lateinit var acm: AssemblerConnectionManager
-    @Inject
-    private lateinit var splss: SecurablePrincipalListStreamSerializer
+
+    private val splss = SecurablePrincipalListStreamSerializer()
```

Comment:
just wanted to double check that there won't be any side effects from removing the `@Inject`?

Reply:
It wasn't initialized with @Inject.. I didn't look really into it, bc it's fairly easy to create a new instance of it. I don't think there should be any side effect here